synapse/federation/transport/server.py

```diff
@@ -1100,7 +1100,7 @@ class FederationGroupsCategoryServlet(BaseFederationServlet):
     """Add/remove/get a category in a group
     """
     PATH = (
-        "/groups/(?P<group_id>[^/]*)/categories/(?P<category_id>[^/]+)"
+        "/groups/(?P<group_id>[^/]*)/categories/(?P<category_id>[^/]+)/?"
```

Comment:
I'm pretty sure we decided adding an optional / here was incorrect

Reply:
Correct, we decided to leave out groups for now.